docs/operation_center/rules_catalog.md

```diff
@@ -0,0 +1,233 @@
+
+The blue square represents a clickable counter of activated rules, it is therefore possible to use the filters related to the effort level by clicking on one or more of them.
+
+### Effort level
+
+The effort level is increasing from Elementary to Master according to two criteria:
+- Effort needed to enable a rule.
+- Risk of false positives.
+Master rules are generic and raise a lot of alerts, which need to be contextualized in a more refined way. Whereas elementary rules require less effort and raise relatively fewer alerts.
+
+### Rules list
+
+The list displays all rules that are available on the Rules Catalog.
+The overview of each rule shows the title, effort level, tags and strategy. It also shows the status of the rule and allows you to enable/disable the rule by clicking on the toggle button on the right side.
+
+### Rules details
+
+On top of the details of a rule, you can see the name, when it was released and last updated.
+There are information about the effort level attributed to the rule, the severity, tags that ease search, related threats.
+It also explains the followed strategy and gives information about false positives, STIIX patterning and data sources.
+To edit an existing rule, click on the gear icon to select entities and assets, or to add a filter alert. 
+
+### Add a rule
+
+To add rule, you can use the + button and fill in the required and optional fields.
+
+- **General definition of the rule:**
+The rule name is mandatory during the creation, it will be used to name the corresponding raised alerts by default. You can add an optional description bellow.
+Select the effort level required and the threats detected with this rule if any, by selecting it on from the MITRE ATT&CK or by using the search bar through key words or drop-down list.
+
+- **Entities & Assets:**
+You can select entities and assets manually or use all of them as a basis to filter the analyzed events.
+Detection rules can be applied to all events, whatever the entity to which they are attached or the assets to which they can be linked, or applied only to some of them. 
+If you choose 'Using all entities' and 'Using all assets', rules will be applied to all events even if new entites or new assets were added since last rule modification. 
+For rules for which some entities or some assets were selected, if new entities or new assets are created, detection will not occur on corresponding events without rule modification.
+
+- **Detection patterns:**
+The signature part corresponds to the rule itself, the detection pattern applied to events. There are two type of rules: CTI or correlation.
+  
+  For CTI rules, you just have to select the source of the indicators: SEKOIA Intelligence Feed is an IOCs feed managed by SEKOIA's Purple Team (indicators present in the SEKOIA.IO Intelligence Center).
```

Comment:
There was a change for this part. For customers who subscribed to Intelligence Center Feed, they can create specific feeds (filtered list of indicators) and use them in CTI rules.